Lib/sqlite3/__main__.py

```diff
@@ -25,22 +25,28 @@ def execute(c, sql, suppress_errors=True):
         for row in c.execute(sql):
             print(row)
     except sqlite3.Error as e:
+        t = theme.traceback
         tp = type(e).__name__
         try:
-            print(f"{tp} ({e.sqlite_errorname}): {e}", file=sys.stderr)
+            tp += f" ({e.sqlite_errorname})"
         except AttributeError:
-            print(f"{tp}: {e}", file=sys.stderr)
+            pass
```

Comment:
I didn't like that the original code was redoing the same thing in case of `AttributeError`. Given that now we're using many more attributes, this looked confusing... what attribute? Hence the change.